app/controllers/albums_controller.rb

```diff
@@ -0,0 +1,57 @@
+class AlbumsController < ApplicationController
+	before_action :set_album, only: [:edit, :show, :update, :destroy, :upvote] 
```

Comment:
Nice use of a before_action.  You could also do and except instead of only like so -> `before_action :set_album , except: [:index, :create, :new]`